pos_no_bank_statement/model/point_of_sale.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# This file is part of OpenERP. The COPYRIGHT file at the top level of
+# this module contains the full copyright notices and license terms.
+
+from openerp import models, api
+
+
+class POSOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.v7
```

Comment:
Because original method in *point_of_sale* is defined using old API. I've had some problems with old-new API compatibilities so I try to write with same API which original method is written. But if you think there would be no problems when decorated with this decorator then I'll rewrite it to new api.

Cheers